src/main/java/nl/workingtalent/bookrental/controller/UserController.java

```diff
@@ -88,6 +120,62 @@ public String userLogin(@RequestBody LoginDto loginDto) {
 		return null;
 	}
 	
+	// TODO: Change return type to status
+	public boolean CheckLoggedInUser(String token)
+	{
+		User loggedInUser = repo.findByToken(token);
+		
+		if (loggedInUser == null) {
+			// TODO: Redirect user to login page
+			return false; // User not logged in
+		}
+		
+		// User is logged in, add success code
+		return true;
+	}
+	
+	// TODO: Change return type to status
+	public boolean CheckUserPermissions(String token)
+	{
+		// Override permissions if it is an automated process 
+		if (token.equalsIgnoreCase("admin")) return true;
+		
+		// Check if user is logged in
+		if (!CheckLoggedInUser(token)) { 
+			return false;
+		}
+		
+		User loggedInUser = repo.findByToken(token);
+		
+		if (!loggedInUser.isAdmin()) {
+			// User has invalid permissions
+			return false;
+		}
+		
+		// User is logged in, and has admin permissions
+		return true;
+	}
```

Comment:
Kan in 1 regel:
!loggedInUser.isAdmin() is een boolean expression dus returned al een True of False